test/entity.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+		<script src="../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+
+		<!-- For IE11 -->
+		<script src="../../lie/dist/lie.polyfill.min.js"></script>
+		<script src="../../whatwg-fetch/fetch.js"></script>
+		<script src="../../wct-browser-legacy/browser.js"></script>
+	</head>
+	<body>
+
+		<script type="module">
```

Comment:
i realize the irony of using something not compatible with ie 11 to test it.  This test is mostly to ensure we're still getting the right behavior in other browsers.  A manual test of 

```
function Entity() {
		if (this.constructor.name === Entity.name) {
			throw new TypeError('Cannot construct Entity instances directly');
		}
}
new Entity();
```
yields ```Cannot construct Entity instances directly``` in ie11

Reply:
Also neat, i guess WCT transpiles the html and all so that it works in ci